spring-ai-core/src/main/java/org/springframework/ai/observation/conventions/SpringAiKind.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.ai.observation.conventions;
+
+/**
+ * Types of Spring AI constructs which can be observed.
+ *
+ * @author Thomas Vitale
+ * @since 1.0.0
+ */
+public enum SpringAiKind {
+
+	// @formatter:off
+
+	CHAT_CLIENT("chat_client"),
```

Comment:
The first two values here are not used yet, I didn't want to make the PR even bigger. We can adopt them in a separate PR.